packages/sdk/src/create.ts

```diff
@@ -39,6 +39,60 @@ const DEFAULT_NODE_REQUIREMENTS = {
 
 export { Libp2pComponents };
 
+const ensureShardingConfigured = (shardInfo: ShardingParams): void => {
+  if (
+    ("shards" in shardInfo && shardInfo.shards.length < 1) ||
+    ("contentTopics" in shardInfo && shardInfo.contentTopics.length < 1)
+  ) {
+    throw new Error(
+      "Missing required configuration options for static sharding or autosharding."
+    );
+  }
+};
+
+/**
+ * Create a Waku node configured to use autosharding or static sharding.
+ */
+export async function createNode(
```

Comment:
To me it seems that we should avoid introduction of a new create function and change the `createLightNode`.

Reply:
Eventually we will change `createLightNode`, but for now we don't want to introduce breaking changes as it was an existing function. If `createLightNode` is used with `options.shardInfo` set, then it behaves almost identical to this new function, with the only difference being that `shardInfo` is optional. If that's satisfactory, then I'm happy to just remove this new function.

Reply:
I think that is enough - but we should make sure to depict it in an example and document it.